instrumentation/dalli/lib/opentelemetry/instrumentation/dalli/instrumentation.rb

```diff
@@ -34,6 +34,7 @@ def add_patches
             ::Dalli::Server.prepend(Patches::Server)
           else
             ::Dalli::Protocol::Binary.prepend(Patches::Server)
+            ::Dalli::Protocol::Meta.prepend(Patches::Server) if defined?(::Dalli::Protocol::Meta)
```

Comment:
The tests currently run on the default protocol. 
https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/06c8de064bec05c787f053967689bcfcd447c840/instrumentation/dalli/test/opentelemetry/instrumentation/dalli/instrumentation_test.rb#L18 

In the latest version, this should default to `binary`. Now that the protocol could be `:binary` or `:meta`, what do you think about updating the tests to run on a `:meta` protocol?